index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Sputnik Realty</title>
+	<link rel="stylesheet" href="main.css">
+</head>
+<body>
+	<header id="topArea">
+		<a href="index.html" id="logo">
+			<h1>Sputnik Realty</h1>
+			<h3>Get your place in Outer Space!</h3>
+		</a>
+		<nav>
+      <div id="topNav">
+  			<ul>
+          <li><a href="index.html" class="selected">Home</li>
+          <li><a href="productPage.html">Products</li>
+  				<li><a href="testimonial.html">Testimonials</li>
+  				<li><a href="contactUs.html">Contact Us</li>
+  			</ul>
+      </div>
+		</nav>
+	</header><!-- /header -->
+	<footer>
+    <div id="bottomNav">
+  		<ul>
+  			<li><a href="index.html" class="selected">Home</li>
+  			<li><a href="productPage.html">Products</li>
+  			<li><a href="testimonial.html">Testimonials</li>
+  			<li><a href="contactUs.html">Contact Us</li>
+  		</ul>
+    </div>
+    <div id="copyright">
+      <p>© 3024 Sputnik Realty</p>
+    </div>
+	</footer>
+</body>
+</html>
```

Comment:
github recommends always having a last blank line at the end of your files- some older browsers and people using linux need it I believe :)